lisp/bazel-build.el

```diff
@@ -24,38 +24,52 @@
 
 ;;; Code:
 
+(defgroup bazel-build 'nil
+  "Package for editing, building, and running code using Bazel."
+  :link '(url-link "https://bazel.build")
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-command "bazel"
+  "Command that should be used to invoke Bazel."
+  :type 'string
+  :group 'bazel-build)
+
 (require 'bazel-util)
```

Comment:
Can you move this require form back to the top of the file? It's just more common to have require forms at the top.

Reply:
Fixed.